robot/web/web_test.go

```diff
@@ -254,12 +254,15 @@ func TestWebWithAuth(t *testing.T) {
 					entityName = options.LocalFQDN
 				}
 
+				// TODO(RSDK-4473) Reenable WebRTC when we figure out why multiple
```

Comment:
I realize I put this comment all over the place, but I do want to remember to reenable WebRTC once we figure out why the deadlock occurs.

Reply:
It does look funny, but I think that's the right call. For a different reason. Assuming getting to the bottom of the bug is slow and this change sits around for a bit: a reader coming across a dozen `rgrpc.Dial` calls where half of them mysteriously use `rpc.WithForceDirectGRPC` is going to be super confused how we choose which type of dial to make. Having comments near each usage is about the best we can do to help provide context.